packages/terra-form-textarea/src/terra-dev-site/doc/example/Invalid.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+
+// eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved, import/extensions
+import Textarea from 'terra-form-textarea/lib/Textarea';
+
+class InvalidExample extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      area: '',
+    };
+    this.handleAreaChange = this.handleAreaChange.bind(this);
+  }
+
+  handleAreaChange(event) {
+    this.setState({ area: event.target.value });
+  }
+
+  render() {
+    return (
+      <Textarea
+        value={this.state.area}
+        isInvalid={this.state.area === ''}
```

Comment:
This isn't displaying the invalid state still. lets try

```
constructor(props) {
    ....
    this.state = {
          isInvalid: true,
    }
}
...
handleAreaChange(event) {
    this.setState({ isInvalid: event.target.value.length < 3 });
}
...
<Textara
    ....
    isInvalid={this.state.isInvalid}
/>
```

Reply:
@JakeLaCombe actually, looking at the text example for invalid state, is there any difference in the view when invalide? https://terra-core-deployed-pr-1938.herokuapp.com/#/tests/terra-form-textarea/form-textarea/invalid-textarea

If not, should we remove this example?

Reply:
I was wondering that too but saw this comment on an earlier PR: https://github.com/cerner/terra-core/pull/982#pullrequestreview-73310815

Reply:
😢 lol I had the same question then as I do now. I don't like it because it feels broken but I guess it's valid!